app/src/main/java/org/jellyfin/androidtv/ui/presentation/CardPresenter.java

```diff
@@ -90,7 +91,10 @@ public void setItem(BaseRowItem m, String imageType, int lHeight, int pHeight, i
                     } else if (imageType.equals(ImageType.THUMB)) {
                         aspect = ImageUtils.ASPECT_RATIO_16_9;
                     } else {
-                        aspect = Utils.getSafeValue(ImageUtils.getImageAspectRatio(itemDto, m.getPreferParentThumb()), ImageUtils.ASPECT_RATIO_7_9);
+                        if (itemDto.getBaseItemType() == BaseItemType.UserView)
```

Comment:
Make this an `else if`

Reply:
Completely changed the method used due to the issue raised below.